src/courses/mappers/06.md

```diff
@@ -0,0 +1,309 @@
+    "groups": [],
+    "status": "loaded",
+    "controls": [
+      ...
+    ],
+    "sha256": "d0506f4b7715bf8b1cd81a8a87ab8efcce41ebeb2b5ec5fcfb23d3bdd136f48c"
+  }
+]
+```
+:::
+
+### 2. <i>**Controls**</i>
+
+::: details Controls Schema
+```json
+controls: [
+  0: {
+    id,
+    title,
+    desc,
+    descriptions,
+    impact,
+    refs,
+    tags,
+    code,
+    source_location,
+    results
+  },
+  ... // More items may exist if there are multiple controls reported per profile
+]
+```
+:::
+
+- `id`: ID of control, used for sorting; should be unique for each unique control.
+- `title`: Title of control.
+- `desc`: Description of the control.
+- `descriptions`: Additional descriptions; usually 'check' and 'fix' text for control.
+- `impact`: Security severity of control. On a scale of 0.1 to 1. This metric is aligned with the [Common Vulnerability Scoring System (CVSS)](https://www.first.org/cvss/).
+- `refs`: References to external control documentation.
+- `tags`: Control tags; typically correlate to existing vulnerability/weakness database (e.g., NIST, CVE, CWE).
+- `code`: Control source code for code preservation.
+- `source_location`: Location of control within source code.
+- `results`: Results substructure (see below).
+
+#### Example
+
+Here's an example of a <i>controls</i> section from a converted GoSec data file:
+
+::: details OHDF Converted GoSec Data
+```json
+"controls": [
+  {
+    "tags": {
+      "nist": ["SI-10"],
+      "cwe": {
+        "id": "22",
+        "url": "https://cwe.mitre.org/data/definitions/22.html"
+      },
+      "nosec": "",
+      "suppressions": "",
+      "severity": "MEDIUM",
+      "confidence": "HIGH"
+    },
+    "refs": [],
+    "source_location": {},
+    "title": "Potential file inclusion via variable",
+    "id": "G304",
+    "desc": "",
+    "impact": 0.5,
+    "results": [
+      ...
+    ]
+  }
+],
+```
+:::
+
+### 3. <i>**Results**</i>
+
+::: details Results Schema
+```json
+results: [
+  0: {
+    status,
+    code_desc,
+    message,
+    run_time,
+    start_time
+  },
+  ... // More items may exist if there are multiple results tested per control
+]
+```
+:::
+
+- `status`: Pass/fail status of test (other statuses exist but are rare).
+- `code_desc`: Test expectations as defined by control.
+- `message`: Demonstration of expected and actual result of test to justify test status.
+- `run_time`: Overall runtime of test.
+- `start_time`: Starting time of test.
+
+#### Example
+
+Here's an example of a <i>results</i> section from a converted Twistlock data file:
+
+::: details OHDF Converted Twistlock Data
+```json
+"results": [
+  {
+    "status": "failed",
+    "code_desc": "Package \"nss-util\" should be updated to latest version above impacted versions [\"*\"]",
+    "message": "Expected latest version of \"nss-util\"\nDetected vulnerable version \"3.67.0-7.el8_5\" of \"nss-util\"",
+    "start_time": "2022-05-18T12:24:22Z"
+  }
+]
+```
+:::
+
+### 4. Additional Structures
```

Comment:
I get that you're trying to not focus as much on this stuff that much which is understandable.  I would find a way to explain that we're in the exec-json context above, and then we can delay talking in-depth about all the rest of its contents aside from profiles til this point.

Reply:
Addressed.